### DISCUSSION TITLE
Any reason for Conductor Monitors being a concrete class

### DISCUSSION THREAD

**Discussion by taoj-action** [Netflix/conductor#2501] [Q&A] [answered]
[Monitors.java](https://github.com/Netflix/conductor/blob/main/core/src/main/java/com/netflix/conductor/metrics/Monitors.java) is a concrete class that creates a hard dependency on [Netflix spectator](https://github.com/Netflix/spectator). It could be more flexible to define an interface instead of those DAOs and let end-user configure different metrics solutions.

**Comment by kishorebanala** [comment, marked as answer]
@taojwmware +1, please send us a PR if you have bandwidth to do this. Thanks!

**Comment by taoj-action** [comment]
I am taking a small step first by cleanup Server dependency by removing unused code.
https://github.com/Netflix/conductor/pull/2503
